app/views/api/v2/smart_class_parameters/main.json.rabl

```diff
@@ -21,4 +21,6 @@ end
 # compatibility
 attribute :omit => :use_puppet_default
 
-attribute :param_class, :as => :puppetclass_name
+node :puppetclass_name do |lk|
```

Comment:
This was kind of intersting. Test failure came back reporting puppetclass_name as 

```
{"id"=>298486374, "name"=>"apache", "created_at"=>"2019-04-10T17:25:22.845Z", "updated_at"=>"2019-04-10T17:25:22.845Z"}
```
rather than `apache`

Reply:
Not sure if this raises any concerns